pkg/virt-handler/vm.go

```diff
@@ -17,109 +17,127 @@ import (
 	"kubevirt.io/kubevirt/pkg/api/v1"
 	"kubevirt.io/kubevirt/pkg/kubecli"
 	"kubevirt.io/kubevirt/pkg/logging"
+
 	"kubevirt.io/kubevirt/pkg/virt-handler/virtwrap"
 	"net/http"
```

Comment:
This isn't related to this commit, but would you mind moving this to the proper import section?